client/test/e2e/patient/record.spec.js

```diff
@@ -0,0 +1,41 @@
+/* jshint expr: true */
+/* global element, by, browser */
+const chai = require('chai');
+const expect = chai.expect;
+
+const FU = require('../shared/FormUtils');
+const components = require('../shared/components');
+const helpers = require('../shared/helpers');
+helpers.configure(chai);
+
+describe('Patient Record', function () {
+
+  const root = '#/patients/';
+  const id = '274c51ae-efcc-4238-98c6-f402bfb39866';
+  
+  const patient = {
+    name : 'Test Patient 2',
+    id : 'TPA 1',
+    hospital_no : '110',
+    age : '25',
+    gender : 'M'
+  };
+  const path = root.concat(id);
+  
+  before(() => helpers.navigate(path));
+  
+  it('downloads and correctly displays patient information', function () {
+    expect(element(by.id('name')).getText()).to.eventually.equal(patient.name);
```

Comment:
Just a note - we could probably make helper to wrap this pretty easily.  Something like `FU.text()`.